docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx

```diff
@@ -640,7 +640,72 @@ selfservice:
 Next, open the login endpoint of the SecureApp and you should see the Facebook
 Login option!
 
-## Google, LinkedIn
+## Google
+
+**Set up on Google Cloud Platform**
+
+1. Create an [OAuth consent screen](https://console.cloud.google.com/apis/credentials/consent)
+2. Create [OAuth 2.0 Client IDs](https://console.cloud.google.com/apis/credentials)
+
+In the new *OAuth 2.0 Client ID*, you will need the `Client ID` and `Client secret`.
+
+**Configuring Kratos**
+
+Create a Jsonnet claims mapper as described in
+[OpenID Connect and OAuth2 Credentials](../concepts/credentials/openid-connect-oidc-oauth2.mdx).
+Save the code in
+`<kratos-directory>/contrib/quickstart/kratos/email-password/oidc.google.jsonnet`.
+
+```json title="contrib/quickstart/kratos/email-password/oidc.google.jsonnet"
+local claims = {
+  email_verified: true
+} + std.extVar('claims');
+
+{
+  identity: {
+    traits: {
+      [if "email" in claims && claims.email_verified then "email" else null]: claims.email,
+      first_name: claims.given_name,
+      last_name: claims.family_name,
+      [if "hd" in claims && claims.email_verified then "hd" else null]: claims.hd,
```

Comment:
What does the hd claim mean?

Reply:
https://developers.google.com/identity/protocols/oauth2/openid-connect#an-id-tokens-payload


`hd | The hosted G Suite domain of the user. Provided only if the user belongs to a hosted domain.`